pandas/tests/indexing/interval/test_interval.py

```diff
@@ -114,6 +114,25 @@ def test_loc_getitem_frame(self):
         with pytest.raises(KeyError, match=r"^\[10\]$"):
             df.loc[[10, 4]]
 
+    def test_string_representation(self):
```

Comment:
move to pandas/tests/indexes/interval/test_formats.py: there are similar tests (search for repr)

Reply:
Done